.env.example

```
@@ -0,0 +1,6 @@
+# DATABASE_URL
+DATABASE_URL=""       # Get connection string from https://cloud.mongodb.com/
+
+# BETTER-AUTH
+BETTER_AUTH_URL=http://localhost:3000
+BETTER_AUTH_SECRET=   # Generate secret via https://better-auth.com/docs/installation
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)